obj-c/getting-started.md

```diff
@@ -5,17 +5,26 @@ Sort: 1
 
 The Mux Objective-C SDK is available either via CocoaPods or you can manually install it. If you have any problems, feel free to get in touch.
 
+## Libraries
+The Mux Objective-C SDK is made of two libraries; the Mux-Stats-Core contains all the core algorithm, and the Mux-Stats-AVPlayer connects iOS/tvOS's AVPlayer or AVPlayerLayer class with the Mux-Stats-Core library. The Mux-Stats-AVPlayer library provides full source code.
+
+## Sample apps
+There are three sample apps in the <a href="https://github.com/muxinc/mux-stats-sdk-avplayer/tree/master/apps" target="_blank">apps</a> folder. DemoApp is an Objective-C iOS demo app, TvDemoApp is a Objective-C tvOS app, and the video-demo is a Swift iOS app. All apps have cocoapod setup for you. Run a `pod install` will download the SDK. Then you can build and run.
+
 ## Installing with CocoaPods
-Modify your `Podfile` to use frameworks by including `use_frameworks!` and then add `pod "Mux-Stats-SDK", "~> 0.0"`. This will use our current release which is version `0.0.3`. There will no be API breaking changes within our `0.0` releases, so you can safely run `pod update`.
+Modify your `Podfile` to use frameworks by including `use_frameworks!` and then add `pod "Mux-Stats-AVPlayer", "~> 0.1.0"` and `pod "Mux-Stats-Core", "~> 2.0.0"`. This will use our current release. There will not be API breaking changes within our `0.0` releases, so you can safely run `pod update`.
```

Comment:
We both pods have to be installed for it to work? I had thought (probably incorrectly) that Mux-Stats-AVPlayer would include Core bundled up in it. If not, that's not a big deal, just wondering.

Reply:
nevermind, apparently both are needed